docs/modules/ROOT/pages/extensions/microprofile-health.adoc

```diff
@@ -0,0 +1,58 @@
+[[microprofile-health]]
+= MicroProfile Health Extension
+
+*Available as of Camel Quarkus version 0.21*
+
+The MicroProfile Health extension bridges the Camel health check API with MicroProfile Metrics & SmallRye Health.
+
+Maven users will need to add the following dependency to their `pom.xml` for this extension.
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel.quarkus</groupId>
+    <artifactId>camel-quarkus-microprofile-health</artifactId>
+    <version>${camel-quarkus.version}</version>
+</dependency>
+------------------------------------------------------------
+
+== Usage
+
+You can register health checks for your applications with the https://camel.apache.org/manual/latest/health-check.html[Camel health check API].
```

Comment:
@zregvart Is this the preferred way to link documents from other Camel guides? I know there is `xref:` - is that one not better for some reason?

Reply:
`xref` would be preferred, with `xref` there's the option of pointing to a specific version of the document and it's not tied to the URL structure of the website.

Reply:
Would the `antora/xref-validator` we run in this repo pass even if the health-check document is in another git repository?